package.json

```diff
@@ -4,11 +4,19 @@
   "private": true,
   "dependencies": {
     "@testing-library/jest-dom": "^5.11.5",
-    "@testing-library/react": "^11.1.0",
-    "@testing-library/user-event": "^12.1.10",
+    "@testing-library/react": "^11.1.1",
+    "@testing-library/user-event": "^12.2.0",
+    "axios": "^0.21.0",
+    "leaflet": "^1.7.1",
+    "mode-sass": "^1.0.0",
```

Comment:
That's probably a typo, delete this if you can't justify why you keep it (it could be malicious code !)

Reply:
About "axios" et "leaflet" we need to use them for call the API and to render the map on the app, except error on my part. Nonetheless, i agree about the others dependencies.